snuba/query/composite.py

```diff
@@ -82,6 +82,9 @@ def set_from_clause(
     ) -> None:
         self.__from_clause = from_clause
 
+    def get_final(self) -> bool:
+        return False
```

Comment:
Is this because composite queries always run with final = false on the top level query?

Reply:
There's no way in the API to actually set FINAL on a JOIN/Sub query and we don't plan to support it. I can't remember if Clickhouse even allows it.

Reply:
Sorry I don't get why we need the method? This method should not exist in this class, who is trying to call it ?